src/pages/SignIn.js

```diff
@@ -20,14 +19,10 @@ const styles = {
 }
 
 const SignIn = () => {
-  const brand = getEnabledBrand()
-
   return h(HeroWrapper, { showMenu: false, showDocLink: true }, [
     div({ style: { maxWidth: 600 } }, [
-      div({ style: { fontSize: 36, color: colors.dark(0.6) } }, ['New User?']),
-      div({ style: { fontSize: 36, marginBottom: '2rem' } }, [`${brand.name} requires a Google Account.`]),
       div({ style: { fontSize: 16, lineHeight: 1.5, marginBottom: '2rem' } }, [
-        `Once you have signed in and completed the user profile registration step, you can start using ${brand.signInName}.`
+        'If you are a new user or returning user, click log-in to continue.'
```

Comment:
It seems unnecessary to say "If you are a new user or returning user", since that covers everyone. I like the previous text better. Also, there's no hyphen in the "Log In" button.

Reply:
I kind of like the short and sweet version. But I really don't care 😂 , I defer to @lstrano-broad